1introduction/tutorial.md

```diff
@@ -0,0 +1,205 @@
+# Codebar
+
+## iOS Clicker Worksheet
+
+This tutorial is an intro to the world of iOS development, and will help you build your first iOS app.
+
+We'll be building a counter app, where the user will press a button and increment the number on the screen.
+
+### Pre-requisites
+
+In order to do iOS development, you will need a Mac.
+
+Download and install Xcode from the App Store. This may take a while — its a big program.
+
+### Creating a Project
+
+1. Open Xcode and click **Create a new Xcode project**.
```

Comment:
If you get an popup asking you to install more components, this is created by Xcode itself.